pkg/templates/upgradeconfig/internal/params/params.go

```diff
@@ -0,0 +1,10 @@
+package params
+
+// Params represents the params accepted by this template.
+type Params struct {
+	StrategyTypeRegex  string
+	MaxPodsUnavailable string
+	MinPodsUnavailable string
+	MaxSurge           string
+	MinSurge           string
```

Comment:
Can you add comments here? Those get populated as descriptions in the docs. Also, if any are required, you should add a "// +required"

Reply:
Done